gradle/license.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'com.github.hierynomus.license'
+
+license {
+    header = rootProject.file('HEADER.txt')
+    strictCheck = true
+    ignoreFailures = true
+    skipExistingHeaders = true
+    mapping {
+        java = 'SLASHSTAR_STYLE'
+        groovy = 'SLASHSTAR_STYLE'
+    }
+    ext.year = Calendar.getInstance().get(Calendar.YEAR)
```

Comment:
[not blocker] I'm curious, should the header contain both dates, e.g. 2007-2016?